doc/api/console.markdown

```diff
@@ -72,15 +72,15 @@ object. This is useful for inspecting large complicated objects. Defaults to
 - `colors` - if `true`, then the output will be styled with ANSI color codes.
 Defaults to `false`. Colors are customizable, see below.
 
-### console.time(timerName)
+### console.time(label)
```

Comment:
@silverwind I don't really care. I changed it after [your suggestion](https://github.com/nodejs/node/pull/3166#discussion_r41427500).
If you want to change it back, just run `git revert` 8c043c1